src/test/java/io/vertx/ext/asyncsql/SQLTestBase.java

```diff
@@ -579,6 +580,22 @@ public void testByteA3Fields(TestContext context) {
       });
     });
   }
+  
+  @Test
+  public void testInvalidInsertStatement(TestContext context) {
+    Async async = context.async();
+
+    client.getConnection(ar -> {
+      ensureSuccess(context, ar);
+      conn = ar.result();
+      conn.updateWithParams("INVALID INSERT", new JsonArray(), ar2 -> {
+        if (ar2.failed() && ar2.cause() instanceof com.github.mauricio.async.db.exceptions.DatabaseException) {
+          async.complete(); 
+        }
```

Comment:
I would add an `else` part, so the test would also complete (but fail instantly) instead of timing out.